chart/values.yaml

```
@@ -14,6 +14,7 @@ prometheus:
   namespace: "monitoring"                    # K8s namespace where prometheus is installed
   namespace_am: "monitoring"                 # K8s namespace where prometheus-alertmanager is installed
   endpoint: "http://prometheus-server.monitoring.svc.cluster.local:80"   # HTTP Endpoint for Prometheus
+  scrapeInterval: 5s
```

We can add a comment to help people populate this value with what they need: 
```suggestion
  scrapeInterval: 5s                        # Prometheus scrape interval. Values is a time duration expressed as a sequence of decimal numbers, each with optional fraction and a unit suffix  such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```